src/User.js

```diff
@@ -0,0 +1,19 @@
+const moment = require('moment');
+
+
```

Comment:
There should only be one empty line before import statements and class signature

Reply:
Gotcha! This has been "taken care of".